script/ide/System/Windows/Controls/Button_p.lua

```diff
@@ -0,0 +1,387 @@
+--[[
+Title: Button
+Author(s): LiXizhi
+Date: 2015/5/7
+Desc: because Button class is too big, we will move private functions to this file.
+use the lib:
+------------------------------------------------------------
+NPL.load("(gl)script/ide/System/Windows/Controls/Button.lua");
+local Button = commonlib.gettable("System.Windows.Controls.Button");
+------------------------------------------------------------
+]]
+local Color = commonlib.gettable("System.Core.Color");
+local Button = commonlib.gettable("System.Windows.Controls.Button");
+
+function Button:initPolygonPensInfo()
+	self:initNormalPens();
+
+--	self.polygon_styles.none = 
+--	{
+--		["pens"] = {
+--			["pen"] = "#ffffff00"
+--		}
+--	};
+end
+
+function Button:initNormalPens()
+	local normal = {
```

Comment:
can you use a class for this to avoid additional table created here.  using class inheritance maybe a better idea